cinema/models.py

```diff
@@ -2,6 +2,20 @@
 from django.db import models
 from django.conf import settings
 
+import os
+import pathlib
+import uuid
+
+from django.utils.text import slugify
+
+
+def create_custom_path(instance, filename):
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
```

Comment:
You can simplify this, you are using f-string + concatination, do you need this?

You can try simply f-string  `f"{slugify(instance.title)}-{uuid.uuid4()}{pathlib.Path(filename).suffix}"